xml/System.Text.Json.Serialization/JsonStringEnumConverter.xml

```diff
@@ -109,9 +109,9 @@ Reading is case insensitive. Writing can be customized by using a <xref:System.T
         <Parameter Name="typeToConvert" Type="System.Type" />
       </Parameters>
       <Docs>
-        <param name="typeToConvert">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="typeToConvert">The type to convert.</param>
+        <summary>Create a converter for the specified type.</summary>
+        <returns>An instance of <see cref="T:System.Text.Json.Serialization.JsonConverter"/> of type T, where T is compatible with <paramref name="typeToConvert"/>.</returns>
```

Comment:
Is that how we generally refer to generics? Why not link to `JsonConverter<T>` rather than "of type T"?

Although, the method is returning the non-generic JsonConverter base type and not the generic `JsonConverter<T>` so the comments in source might be misleading.

@steveharter - what do you think this should be doc'd as? Why are we talking about the T here for a non-generic type?